Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.8.5
 
 ARG MYBRANCH=master
+ARG COMMIT=""
```

Comment:
Maybe default this to something like "Unknown" if it's not provided in the `docker build`

Reply:
good idea, done.